features/samples.feature

```diff
@@ -9,9 +9,9 @@ Feature: Basic smell detection
     Then the exit status indicates smells
     And it reports:
     """
-    inline.rb -- 43 warnings:
+    inline.rb -- 44 warnings:
       CompilationError has no descriptive comment (IrresponsibleModule)
-      File has no descriptive comment (IrresponsibleModule)
+      Dir has no descriptive comment (IrresponsibleModule)
```

Comment:
I have no idea why this wasn't failing before but in fact this feature failure was legit. If you check out spec/samples/inline.rb you can see that File DOES have a descriptive comment: https://github.com/troessner/reek/blob/develop/spec/samples/inline.rb#L682 at the end whereas Dir does not have one: https://github.com/troessner/reek/blob/develop/spec/samples/inline.rb#L684 neither at the beginning, nor at the end.
So if you just copy the Dir and File declaration (just the class name and the "end" and the comments) to your editor, save that as a separate file and run reek on it it will rightfullly report the same.